### PR TITLE
Adding x-ms-pageable tag for configuration list command.

### PR DESCRIPTION
**Pull request by Prasanna-Padmanabhan**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/advisor/resource-manager/Microsoft.Advisor/stable/2017-04-19/advisor.json [hunk 1]
```diff
@@ -41,6 +41,9 @@
           }
         },
         "deprecated": false,
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
         "x-ms-examples": {
```
Comment (olydis) [inline specification/advisor/resource-manager/Microsoft.Advisor/stable/2017-04-19/advisor.json]: shouldn't `Configurations_ListByResourceGroup` be marked the same way then? 🙂  Note that this is an SDK breaking change, but I assume nothing has been released so far?

File: specification/advisor/resource-manager/readme.md [hunk 2]
```diff
@@ -89,6 +89,7 @@ python:
   payload-flattening-threshold: 2
   namespace: azure.mgmt.advisor
   package-name: azure-mgmt-advisor
+  package-version: 1.0.1
```
Comment (olydis) [inline specification/advisor/resource-manager/readme.md]: @lmazuel cool?

**Comment by Prasanna-Padmanabhan** [comment]
The resource group will only contain one configuration. The subscription can have one configuration for itself and one for each of its resource groups, hence the need for paging.

This is a bug that we realized when building the CLI module, so it needs to be fixed to reflect how the API is behaving.

File: specification/advisor/resource-manager/readme.md (hunk 2 above)
Comment (lmazuel) [reply specification/advisor/resource-manager/readme.md]: @olydis yes! I reviewed these changes offline, with actual RestAPI look. This is indeed an expected bugfix.

File: specification/advisor/resource-manager/Microsoft.Advisor/stable/2017-04-19/advisor.json (hunk 1 above)
Comment (lmazuel) [reply specification/advisor/resource-manager/Microsoft.Advisor/stable/2017-04-19/advisor.json]: Definitly. @Prasanna-Padmanabhan please follow @olydis guidelines. Thanks!!

Comment (Prasanna-Padmanabhan) [reply specification/advisor/resource-manager/Microsoft.Advisor/stable/2017-04-19/advisor.json]: The resource group will only contain one configuration. The subscription can have one configuration for itself and one for each of its resource groups, hence the need for paging.

Comment (olydis) [reply specification/advisor/resource-manager/Microsoft.Advisor/stable/2017-04-19/advisor.json]: If only one page is expected from that operation, you can mark it with
``` JSON
   "x-ms-pageable": {
          "nextLinkName": null
   },
```
The UX of the generated SDK will be better, as the useless nesting `value` property of the return value is hidden from the user.

Comment (Prasanna-Padmanabhan) [reply specification/advisor/resource-manager/Microsoft.Advisor/stable/2017-04-19/advisor.json]: I see, OK, added that decoration per your suggestion.

**Review by olydis — APPROVED** [review]

**Pull request merged**